src/tools/vs-code.md

```diff
@@ -41,56 +41,65 @@ There are a couple ways to create a new project.
 
- 1. Open the Command Palette
-    (`Ctrl`+`Shift`+`P` (`Cmd`+`Shift`+`P` on macOS)).
- 1. Select the **Flutter: New Project** command and press `Enter`.
- 1. Select **Application** and press `Enter`.
- 1. Select a **Project location**.
- 1. Enter your desired **Project name**.
+1. Open the Command Palette.
+
+   - On Windows or Linux,
+     press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>.
+   - On macOS,
+     press <kbd>Cmd</kbd> + <kbd>Shift</kbd>+<kbd>P<kbd>.
+
+1. Tyoe `flutter`.
+1. Select the **Flutter: New Project**.
+1. Press <kbd>Enter</kbd>.
+1. Select **Application**.
+1. Press <kbd>Enter</kbd>.
+1. Select a **Project location**.
+1. Enter your desired **Project name**.
 
 ### Opening a project from existing source code
 
 To open an existing Flutter project:
 
- 1. Click **File > Open** from the main IDE window.
- 1. Browse to the directory holding your existing
-    Flutter source code files.
- 1. Click **Open**.
+1. From the main IDE window, go to **File** > **Open**.
+1. Browse to the directory holding your existing
+   Flutter source code files.
+1. Click **Open**.
 
 ## Editing code and viewing issues
 
 The Flutter extension performs code analysis that
 enables the following:
 
-* Syntax highlighting
-* Code completions based on rich type analysis
-* Navigating to type declarations
-  (**Go to Definition** or `F12`),
+- Syntax highlighting
+- Code completions based on rich type analysis
+- Navigating to type declarations
+  (**Go to Definition** or press <kbd>F12</kbd>),
   and finding type usages
-  (**Find All References** or `Shift`+`F12`)
-* Viewing all current source code problems
-  (**View > Problems** or `Ctrl`+`Shift`+`M`
-  (`Cmd`+`Shift`+`M` on macOS))
+  (**Find All References** or press <kbd>Shift</kbd> + <kbd>F12</kbd>)
+- Viewing all current source code problems
+  (**View** > **Problems**
+  or <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>M</kbd>
+  or <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>M</kbd> on macOS)
   Any analysis issues are shown in the Problems pane:<br>
   ![Problems pane]({{site.url}}/assets/images/docs/tools/vs-code/problems.png){:.mw-100.pt-1}
 
 ## Running and debugging
 
 {{site.alert.note}}
-  You can debug your app in a couple of ways.
+You can debug your app in a couple of ways.
 
-  * Using [DevTools][], a suite of debugging and profiling
-    tools that run in a browser. 
-  * Using VS Code's built-in debugging features,
-    such as setting breakpoints.
+- Using [DevTools][], a suite of debugging and profiling
+  tools that run in a browser.
+- Using VS Code's built-in debugging features,
+  such as setting breakpoints.
 
-  The instructions below describe features available in VS Code.
-  For information on using launching DevTools, see
-  [Running DevTools from VS Code][] in the [DevTools][] docs.
+The instructions below describe features available in VS Code.
+For information on using launching DevTools, see
+[Running DevTools from VS Code][] in the [DevTools][] docs.
```

Comment:
Our current style is to indent alerts two spaces.

Reply:
Yes, and Markdownlint shows this as violating [MD007](https://github.com/DavidAnson/markdownlint/blob/main/doc/md007.md) because of the list. I dedented because of that warning.